manifests/alertmanager.pp

```diff
@@ -0,0 +1,242 @@
+# Class: prometheus::mysqld_exporter
```

Comment:
What's the reason for renaming `prometheus::alert_manager` to `prometheus::alertmanager`? This is a breaking change.